Source/WebCore/rendering/RenderMenuList.cpp

```diff
@@ -116,21 +116,36 @@ void RenderMenuList::setInnerRenderer(RenderBlock& innerRenderer)
 
 void RenderMenuList::adjustInnerStyle()
 {
+    auto isHorizontalWritingMode = style().isHorizontalWritingMode();
+
     auto& innerStyle = m_innerBlock->mutableStyle();
     innerStyle.setFlexGrow(1);
     innerStyle.setFlexShrink(1);
     // min-width: 0; is needed for correct shrinking.
-    innerStyle.setMinWidth(Length(0, LengthType::Fixed));
+    if (isHorizontalWritingMode)
+        innerStyle.setMinWidth(Length(0, LengthType::Fixed));
+    else
+        innerStyle.setMinHeight(Length(0, LengthType::Fixed));
```

Comment:
it looks like we have RenderStyle::logicalMin/MaxWidth(), maybe we should have a setLogicalMin/MaxWidth too?

Reply:
Added.